doc/devel/triage.rst

```diff
@@ -0,0 +1,214 @@
+
+.. _bug_triaging:
+
+Bug triaging and issue curation
+================================
+
+The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_
+is important to the communication in the project: it is where users to
+make feature requests and report bugs, developers identify major
+projects to work on and discuss priorities. For this reason, it is
+important to curate the issue list, adding labels to issues and
+closing issues that are not necessary.
+
+
+Triaging issues does not require any particular expertise in the
+internals of Matplotlib, is extremely valuable to the project, and we
+welcome anyone to participate in issue triage!  However, people who
+are not part of the Matplotlib organization do not have `permissions
+to change milestones, add labels, or close issue
+<https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization>`_.
+If you do not have enough permissions do to a step, please leave a
+comment tagging ``@matplotlib/triage_team`` with your recommendations.
+
+Working on issues to improve them
+---------------------------------
+
+Improving issues increases their chances of being successfully resolved.
+Guidelines on submitting good issues can be found :ref:`here
+<submitting-a-bug-report>`.
+A third party can give useful feedback or even add
+comments on the issue.
+The following actions are typically useful:
+
+  - documenting issues that are missing elements to reproduce the problem
+    such as code samples
+
+  - suggesting better use of code formatting
+
+  - suggesting to reformulate the title and description to make them more
+    explicit about the problem to be solved
+
+  - linking to related issues or discussions while briefly describing
+    how they are related, for instance "See also #xyz for a similar
+    attempt at this" or "See also #xyz where the same thing was
+    reported" provides context and helps the discussion
+
+  - verifying that the issue is reproducible
+
+  - classify the issue as a feature request, a long standing bug or a
+    regression
+
+.. topic:: Fruitful discussions
+
+   Online discussions may be harder than it seems at first glance, in
+   particular given that a person new to open-source may have a very
+   different understanding of the process than a seasonned maintainer.
+
+   Overall, it is useful to stay positive and assume good will. `The
+   following article
+   <http://gael-varoquaux.info/programming/technical-discussions-are-hard-a-few-tips.html>`_
+   explores how to lead online discussions in the context of open source.
+
+
+Triage Team
+-----------
+
+
+If you would like to join the triage team:
+
+1. Correctly triage 2-3 issues.
+2. Ask someone with commit rights to recommend you to the triage team
```

Comment:
please link here to the list of people w/ commit rights (which separately, a lot of folk with commit rights aren't active)  & possibly the default person they should reach out to (which is why my bias is towards a form) 'cause I can imagine folk not wanting to bother someone w/ commit rights - if I was in this boat when I started contributing, I would have asked Ryan 'cause I already knew him. Possible add another sentence here is "for example, one of the contributors that you have interacted with on an issue" to make it clear the contributors are on board w/ it & who they should be asking.